packages/cedar/src/Chart.ts

```diff
@@ -14,11 +14,16 @@ export default class Chart {
   private _cedarSpecification: any
   private _data: any[]
   private _overrides: any
+  private _container: string
 
-  constructor(options: any) {
+  constructor(container: string, options: any) {
```

Comment:
`container` should be a `string` or element. I think this is the right syntax: `container: string | HTMLElement`

Also I think b/c we set the types on the `private` vars above, we don't kneed to set them here in the constructor.